spec/migrations/20160911130449_add_value_serialized_to_custom_attribute_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_migration
+describe AddValueSerializedToCustomAttribute do
+  migration_context :up do
+    it 'migrate current values to serialized values correctly' do
+      FactoryGirl.create(:custom_attribute, :name => 'example', :value => "foo")
```

Comment:
Additionally use the migration_stub to create the record to avoid loading the "real" CustomAttribute model